package.json

```diff
@@ -1,9 +1,11 @@
 {
   "dependencies": {
+    "bluebird": "^3.5.0",
```

Comment:
can we not have 20 promise libraries in this project ? ES6 has promises, then we have RSVP already, and you are adding Bluebird. Quintessential nodejs dependency bucket ?